linode/domains/datasource_test.go

```diff
@@ -18,40 +18,42 @@ func TestAccDataSourceDomains_basic(t *testing.T) {
 
 	domainName := acctest.RandomWithPrefix("tf-test") + ".example"
 
-	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { acceptance.PreCheck(t) },
-		ProtoV5ProviderFactories: acceptance.ProtoV5ProviderFactories,
-		Steps: []resource.TestStep{
-			{
-				Config: tmpl.DataBasic(t, domainName),
-				Check: resource.ComposeTestCheckFunc(
-					acceptance.CheckResourceAttrGreaterThan(resourceName, "domains.#", 1),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.id"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.domain"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.type"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.description"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.status"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.tags.0"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.soa_email"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.retry_sec"),
-					resource.TestCheckResourceAttrSet(resourceName, "domains.0.expire_sec"),
-				),
+	acceptance.RunTestRetry(t, 2, func(tRetry *acceptance.TRetry) {
```

Comment:
Saw this test fail occasionally during local testing, just thought I add a retry here for stability